### PR TITLE
Level markdown cleanup: remove external CSS links

### PR DESCRIPTION
**Pull request by joshlory**
Initial cleanup of PD markdown levels.  I've begun removing links to the following stylesheet from `External` levels:

``` html
<link rel="stylesheet" type="text/css" href="//code.org/curriculum/docs/web/k5online.css">
```

I've started cleaning up the styles to match other markdown levels (mostly updating the headers to be `#`, `##` instead of `###`, `####`).

Sample level before:
markdown-css-cleanup-before

Sample level after:
markdown-css-cleanup-after

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -73,6 +73,7 @@ h1 {
 }
 
 h2 {
+  margin: 25px 0 10px 0;
```
Comment (Hamms) [inline dashboard/app/assets/stylesheets/application.css.scss:76]: this seems like it has the potential to have very broad implications. Where else do we use `h2`s?

File: dashboard/config/scripts/csppd2_lesson_1_reflection.external [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+<img src="https://s3.amazonaws.com/uploads.hipchat.com/65395/734864/kROPqudAyIm2zdw/Screen%20Shot%202015-07-17%20at%209.49.21%20PM.png" class="level-image">
+
+## Reminder: Accessing Forums
 *Each lesson plan includes a link to a dedicated forum thread in the "Materials and Prep" section on the right side of the first page (shown on the right). You can always access the thread for a given lesson by following the link inside the lesson*. 
-<br><br><br><br>
-<a href="http://www.google.com/url?q=http%3A%2F%2Fforum.code.org%2Ft%2Flesson-1-the-impact-of-innovation%2F586%2F1&sa=D&sntz=1&usg=AFQjCNG6jca0p_Sge_wiKDUWG-2nedKqUQ" target="blank"><button class="plc">Forum Thread for Lesson 1 >></button></a>
-<br><br>
->#### Assignment: Post to forum with thoughts on Lesson 1
+
+<h4><a href="http://www.google.com/url?q=http%3A%2F%2Fforum.code.org%2Ft%2Flesson-1-the-impact-of-innovation%2F586%2F1&sa=D&sntz=1&usg=AFQjCNG6jca0p_Sge_wiKDUWG-2nedKqUQ" target="blank">Forum Thread for Lesson 1 <i class="fa fa-external-link-square" /></a></h4>
```
Comment (Hamms) [inline dashboard/config/scripts/csppd2_lesson_1_reflection.external]: why `<h4>` and not `####`?

File: dashboard/config/scripts/csppd3_u2_chunk_2_concepts.external [hunk 3]
```diff
@@ -2,40 +2,34 @@ name 'CSPPD3-u2 chunk 2 concepts'
...
 
 >- Caesar Cipher 
 - Random Substitution Cipher
 - Vigenere Cipher
 
-#### Using Hard Problems to Secure Your Information
- This section of lessons starts with a series of unplugged activities that explore a class of computer science problems that are hard for computers (and humans) to solve. The are called ‘computationally hard’ problems, and they are the basis of the encryption method we use and study here. 
-<br><br>
+## Using Hard Problems to Secure Your Information
+This section of lessons starts with a series of unplugged activities that explore a class of computer science problems that are hard for computers (and humans) to solve. The are called ‘computationally hard’ problems, and they are the basis of the encryption method we use and study here.
```
Comment (Hamms) [inline dashboard/config/scripts/csppd3_u2_chunk_2_concepts.external:22]: Given that we're replacing `&` with `&amp;`, do we also want to replace these special curly quotes with regular ones?

**Comment by Hamms** [comment]
couple open questions, but generally LGTM

**Comment by tanyaparker** [comment]
Have we also been updating levelbuilders that they should no longer use k5online.css?

**Comment by joshlory** [comment]
I had a brief conversation with Kiki about this stylesheet:

screen shot 2016-07-19 at 12 16 54 pm

Longer term it would be good to set up better guidelines for level content.  Should there be a review process for new level content?  Certainly not for every level, but maybe worth checking at a high level that new content will fit into the existing ui-ux scheme.

Brendan had a lot of good usability feedback (e.g. about usage of images of text) that would be good to incorporate.

File: dashboard/app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (joshlory) [inline dashboard/app/assets/stylesheets/application.css.scss:76]: Definitely a risk here, but we need to be able to modify these base styles in order to align with Pegasus.  Hopefully any component that doesn't want the "default" site-wide margin on `<h2>` has already set the correct margins.  If not I'll have to fix any issues as they arise 😄.

File: dashboard/config/scripts/csppd3_u2_chunk_2_concepts.external (hunk 3 above)
Comment (joshlory) [inline dashboard/config/scripts/csppd3_u2_chunk_2_concepts.external:22]: I can, but without a process to auto-convert them they'll likely be re-introduced.

Open to suggestions for how to solve some of these common lint issues!  ("Smart" quotes, trailing whitespace, etc.)  There are markdown lint tools, but they may be overkill.

File: dashboard/config/scripts/csppd2_lesson_1_reflection.external (hunk 2 above)
Comment (joshlory) [inline dashboard/config/scripts/csppd2_lesson_1_reflection.external]: Fixed in 70a469afb82d49da9c809ba84e9a3708eaf8652e.

**Comment by joshlory** [comment]
Amended the previous commit.

**Pull request merged**